daml-foundations/daml-ghc/ghc-compiler/src/DA/Daml/GHC/Compiler/Options.hs

```diff
@@ -64,6 +64,8 @@ data Options = Options
     -- ^ Whether to enable debugging output
   , optGhcCustomOpts :: [String]
     -- ^ custom options, parsed by GHC option parser, overriding DynFlags
+  , optScenarioService :: Bool
+    -- ^ disable scenario service when False
```

Comment:
It’s probably worth pointing out that setting it to `True` does not enable it.

Reply:
:+1: